Packs/SysAid/Integrations/SysAid/SysAid.py

```diff
@@ -631,6 +723,10 @@ def service_record_list_command(client: Client, args: Dict[str, Any]) -> Command
 
     custom_fields_keys = argToList(args.get('custom_fields_keys'))
     custom_fields_values = argToList(args.get('custom_fields_values'))
+
+    # SysAid expects the timestamp to be provided as `timestamp,timestamp`, but we are splitting on ,
+    # This breaks the inputs. Instead we are asking for `timestamp-timestamp` as input, then convert the - to ,
+    custom_fields_values = [re.sub(r"-", ",", value, 0, re.MULTILINE) for value in custom_fields_values]
```

Comment:
```suggestion
    custom_fields_values = [value.replace("-", ",") for value in custom_fields_values]
```
Is it always wanted?

Reply:
No, but it is required when timeframes are requested.